src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -333,17 +278,14 @@ public int compare(final ProTerritory t1, final ProTerritory t2) {
     });
 
     // Log prioritized territories
-    for (final ProTerritory patd : prioritizedTerritories) {
+    for (final ProTerritory patd : attackOptions) {
```

Comment:
Problem with toString() is you can get many different needs for it and that can lead to issues. Nothing wrong with "toDebugString()", and or creating a DebugStringFactory that will do it for you: `DebugFormatter.toAttackValueDebugString( patd)`